src/components/app-content-renderer/styles/panels.scss

```diff
@@ -112,31 +112,39 @@
   .second-panel {
     background-color: var(--pf-global--BackgroundColor--100);
     color: var(--pf-global--Color--100);
-
-    @media screen and (min-width: 1450px) {
-      .pf-l-grid {
-        max-width: 1160px;
-        // grid-template-columns: repeat(auto-fill, calc(100% / 3 - var(--pf-c-description-list--ColumnGap)));
-      }
+    padding: var(--pf-global--spacer--xl) var(--pf-global--spacer--lg) var(--pf-global--spacer--lg) var(--pf-global--spacer--lg);
+   
+    .container {
+      max-width: 1100px;
```

Comment:
On 4K monitors this will be too small, can you add a note here to refactor this?

Reply:
@karelhala The overall width was set by the UXD design. It was never intended as a fluid layout.